Resources/views/Form/fields.html.twig

```diff
@@ -336,11 +337,11 @@
   {%- endif -%}
   {%- set widget_attr = {} -%}
   {%- if help is not empty -%}
-    {%- set widget_attr = {attr: {'aria-describedby': id ~"_help"}} -%}
+    {%- set widget_attr = {attr: {'aria-describedby': id ~ "_help"}} -%}
   {%- endif -%}
   <{{ element|default('div') }} class="form-group">
-    {{- form_label(form) -}}
-    {{- form_widget(form, widget_attr) -}}
+  {{- form_label(form) -}}
+  {{- form_widget(form, widget_attr) -}}
   </{{ element|default('div') }}>
```

Comment:
same, waarom is `<` weg?

Reply:
Idem, zijn enkel de spaties. Geen idee waarom Github het in het rood zet